_posts/2021-04-14-virtual-workshop-announcement.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: post
+title: 2021 US-RSE Virtual Workshop
+tags: [events, virtual workshop]
```

Comment:
Should this perhaps just be workshop?
```suggestion
tags: [events, workshop]
```

Reply:
Maybe we can leave virtual workshop and workshop? it might be a good tag in the future to look how many virtual workshops we had

Reply:
That would work for me! But if we've had a virtual workshop already, we should update the previous ones. Also I noticed we have two tags for workshop, both workshop and workshops (we probably should have just one?)

Reply:
If you can tell me which of our previous events were virtual I can put in a PR to fix up the previous records.

Reply:
I think it was just our April workshop last year yet https://us-rse.org/events/2020/2020-04-virtual-workshop

Reply:
Thank you! See PR https://github.com/USRSE/usrse.github.io/pull/438.